server/src/services/cleanUserInput.js

```diff
@@ -0,0 +1,10 @@
+const cleanUserInput = formInput => {
```

Comment:
I see this was added, but is not used? Looks like the existing service you have cleanBuildForm does the same thing. As a suggestion, cleanUserInput can be used for any resource, whereas the cleanBuildForm indicates it is meant only for build resources

Reply:
deleted